api/prisma/schema.prisma

```diff
@@ -86,6 +86,8 @@ model Publication {
     funders                            Funders[]
     fundersStatement                   String?
     affiliations                       Affiliations[]
+    confirmNoAffliations               Boolean?
+    affiliationsStatus                 Boolean?               @default(true)
```

Comment:
Since this is holding a boolean, I think 'hasAffiliations' would have been more appropriate.
Status usually refers to a string like: ACTIVE, INACTIVE, LOCKED, LIVE, DRAFT and so on...